File: tags/release-1.0.0/script/test_frontend_webservice.py
#!/usr/bin/env python

import os
import sys

pcdDir = '../pcd'

paths = [ pcdDir, pcdDir + '/vendor', pcdDir + '/modules' ]
for item in paths:
    sys.path.append(item)

basePath = os.path.abspath("../")

# ==============================================================================

from conf import runtimeConfig

config = runtimeConfig(pcdDir)
token = config.token
remote = "%s%s" % (config.getElement('base/url'),
                   config.getElement('webservice/frontendUrl'))

import xmlrpclib

server = xmlrpclib.ServerProxy(remote)

# ==============================================================================

print "---------------------------------------------------------------------------------"
print "feed update get"
print "---------------------------------------------------------------------------------"

print server.feed_update_get({ 'token' : token, 'total' : 3 })

File: tags/release-1.1.0/script/test_frontend_webservice.php
#!/usr/bin/env php
<?php

require "../application/console.php";

$ws = new L_WebService();
$token = B_Registry::get('webservice/token');

echo "* feed_update_get\n";
$r = $ws->feed_update_get(array('token' => $token));
print_r($r);
